shell/platform/darwin/macos/framework/Headers/FlutterAppDelegate.h

```diff
@@ -10,6 +10,30 @@
 #import "FlutterAppLifecycleDelegate.h"
 #import "FlutterMacros.h"
 
+/**
+ * A protocol to be implemented by the `NSApplicationDelegate` of an application to enable the
+ * Flutter framework and any Flutter plugins to register to receive application life cycle events.
+ *
+ * Implementers should forward all of the `NSApplicationDelegate` methods corresponding to the
+ * handlers in FlutterAppLifecycleDelegate to any registered delegates.
+ */
+FLUTTER_DARWIN_EXPORT
+@protocol FlutterAppLifecycleProvider <NSObject>
```

Comment:
This change isn't strictly necessary, but it makes us match iOS, and it makes the engine code that uses these methods cleaner.